internal/controller/gpunode_controller.go

```diff
@@ -464,7 +464,7 @@ func (r *GPUNodeReconciler) createHypervisorPod(
 		newPod.Spec.Tolerations = []corev1.Toleration{}
 	}
 	newPod.Spec.Tolerations = append(newPod.Spec.Tolerations, corev1.Toleration{
-		Key:      string(corev1.TaintEffectNoSchedule),
+		Key:      string(corev1.TaintEffectPreferNoSchedule),
```

Comment:
key should be `constants.NodeUsedByTaintKey`
```suggestion
		Key:      string(constants.NodeUsedByTaintKey),
```